Tests/LLVMTests/ConstantSpec.swift

```diff
@@ -89,6 +89,29 @@ class ConstantSpec : XCTestCase {
       // FLOATINGCONST-NEXT: }
       module.dump()
     })
+
+    XCTAssert(fileCheckOutput(of: .stderr, withPrefixes: ["STRUCTCONST"]) {
+      // STRUCTCONST: ; ModuleID = '[[ModuleName:ConstantTest]]'
+      // STRUCTCONST-NEXT: source_filename = "[[ModuleName]]"
+      let module = Module(name: "ConstantTest")
+      let builder = IRBuilder(module: module)
+      // STRUCTCONST: define void @main() {
+      let main = builder.addFunction("main",
+                                     type: FunctionType(argTypes: [],
+                                                        returnType: VoidType()))
+
+      let constant = StructType(elementTypes: [IntType.int64])
+        .constant(values: [Int64(42).asLLVM()])
```

Comment:
Requiring this call to `.asLLVM()` is unfortunate. Can we get rid of this, somehow? Maybe `Constant(int: ...)`?